### PR TITLE
Remove operation id prefix

### LINKED ISSUE #866: Requires remove operationid prefix
When use smallrye openapi operationid strategy CLASS_METHOD,operationId fields take the form of "class_name_method". For example, "User_findAll".
we need remove operationid prefix that already exists in the OpenAPI Generator.

### PR DESCRIPTION
**Pull request by yuhaibohotmail**
- [x]  You have read the [contributors guide](CONTRIBUTING.md)
- [x]  Your code is properly formatted according to [our code style](CONTRIBUTING.md#code-style)
- [x]  Pull Request title contains the target branch if not targeting main: `[0.9.x] Subject`
- [x] Pull Request contains link to the issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x]  Pull Request does not include fixes for issues other than the main ticket

Fix #866

When use smallrye openapi operationid strategy CLASS_METHOD,operationId fields take the form of "class_name_method". For example, "User_findAll". We want remove operationId prefix (e.g. User_findAll=> findAll)

In this PR, we add a config properties remove-operation-id-prefix,to delete class name from operationId.  The properties are passed to the openapi-generator, and the specific functions are executed by the openapi-generator.

https://github.com/OpenAPITools/openapi-generator/blob/4c5a57fe7bae59483bb8fa9218be8dc4f82e1bda/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L5638

### CODE REVIEW

**Review by fjtirado — APPROVED** [review]

**Review by ricardozanini — CHANGES_REQUESTED** [review]
I need help finding what this PR is doing. I'm only seeing an introduction of a property to the generator. What do you do with it after? Or is this something that the underlying generator already takes care of? If so, please add a reference.
Can you also please add some tests under the `integration-tests` module?

File: client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java [hunk 1]
```diff
@@ -5,16 +5,7 @@
...
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.BASE_PACKAGE;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.DEFAULT_SECURITY_SCHEME;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.EXCLUDE;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.INCLUDE;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.INPUT_BASE_DIR;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.MODEL_NAME_PREFIX;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.MODEL_NAME_SUFFIX;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.TEMPLATE_BASE_DIR;
-import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.VALIDATE_SPEC;
+import static io.quarkiverse.openapi.generator.deployment.CodegenConfig.ConfigName.*;
```
[the remark this answers is not in the document]
Comment (yuhaibohotmail) [reply client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java]: Already corrected.

File: docs/modules/ROOT/pages/includes/getting-started.adoc [hunk 2]
```diff
@@ -88,6 +88,13 @@ quarkus.openapi-generator.codegen.spec.petstore_json.model-name-suffix=CustomMod
 quarkus.openapi-generator.codegen.spec.petstore_json.model-name-prefix=CustomModelPrefix
 ----
 
+You can customize the name of generated method. To do that, you must define the following properties:
```
[the remark this answers is not in the document]
Comment (yuhaibohotmail) [reply docs/modules/ROOT/pages/includes/getting-started.adoc]: we add a config properties remove-operation-id-prefix,to delete class name from operationId. The properties are passed to the openapi-generator, and the specific functions are executed by the openapi-generator.

**Comment by yuhaibohotmail** [comment]
> I need help finding what this PR is doing. I'm only seeing an introduction of a property to the generator. What do you do with it after? Or is this something that the underlying generator already takes care of? If so, please add a reference. Can you also please add some tests under the `integration-tests` module?

Thanks for the review.
I only add the property , the functions are executed by the openapi-generator.
This is a function of openapi-generator. Do I need to write an integration test?

**Comment by ricardozanini** [comment]
@yuhaibohotmail, thanks for clarifying it. You can use the `deployment` module tests to test it or add a new `integration-test` module.

**Comment by yuhaibohotmail** [comment]
> @yuhaibohotmail, thanks for clarifying it. You can use the `deployment` module tests to test it or add a new `integration-test` module.
@ricardozanini , we have the new `integration-test` module.

**Pull request closed without merging**